core/src/main/scala/org/apache/spark/SparkStatusTracker.scala

```diff
@@ -104,4 +106,34 @@ class SparkStatusTracker private[spark] (sc: SparkContext) {
       }
     }
   }
+
+  /**
+   * Returns a list of all known executors, represented by string with format: "host:port"
+   */
+  def getExecutors(): Array[String] = {
```

Comment:
If I just want a list of executors, why shouldn't I be able to get them? I think it makes sense to have a more detailed version (maybe replace the following 2), but having a simple one that returns just the list of executors seem to make sense too.